src/content/docs/configuration/providers.mdx

```diff
@@ -20,114 +20,221 @@ Daytona abstracts the management and deployment of Workspaces into 2 related con
+        │ v0.0.0
+    ```
+
+    ```text
+        Provider docker-provider has been successfully updated
+    ```
+
 ## Managing Targets
 
 ### Set a Target
 
-You can set a new Target for Daytona to use when managing Workspaces.
+Daytona allows you to set a Target to use when managing Workspaces.
 
 **Prerequisites**
 
-- A working installation of Daytona.
-- At least one Provider installed.
+- **At least [one Provider](#managing-providers) installed.**
+
+    Providers are essential for managing and deploying your Workspaces, so make sure you have configured at least one.
 
 **Procedure**
 
-1. Execute the following command:
+1. Run the following command to set a Target:
 
     ```shell
     daytona target set
     ```
 
 2. Select the appropriate Provider for the environment you want to deploy to.
+
+    ```text
+        Choose a provider
+        1 item
+        ===
+        │ docker-provider
+        │ v0.0.0
+    ```
+
 3. Select `New Target`.
 4. Enter a name for your Target.
 5. Enter the appropriate configuration options when prompted.
-6. Click `Enter` to add the Target.
 
-### Remove a Target
+    ```text
+        Remote Password
+        >
 
-You can remove an existing target.
-Once a Target is removed, you will not be able to manage or deploy Workspaces without adding another Target with the same configuration.
+        Remote Port
+        >
 
-**Prerequisites**
+        Remote User
+        Note: non-root user required
+        >
+
+        Sock Path
+        > /var/run/docker.sock
+
+        Workspace Data Dir       
+        The directory on the remote host where the workspace data will be stored
+        > /tmp/daytona-data
+
+        ┃ Remote Private Key Path
+            daytona_config
+            known_hosts
+            Custom path
+            None
+    ```
+
+6. Click `Enter` to confirm setting the Target.
 
-- A working installation of Daytona.
-- At least one configured Target.
+    ```text
+        Target set successfully
+     ```
+
+### List Targets
+
+Daytona allows you to keep track of your Targets by listing all previously created Targets with their details.
+
+1. Run the following command to list currently set Targets:
+
+   ```shell
+   daytona target list
+   ```
+
+   Upon running this command, Daytona will display a list of your Targets with their details. You will be able to see the Target name, the Provider it is connected to, and its configured options.
+
+   ```text
+    Target      Provider            Options
+    ───────────────────────────────────────────────────────────────────────
+    MyTarget    docker-provider     {
+                                        "Sock Path": "/var/run/docker.sock"
+                                    }
+   ```
+
+### Delete a Target
+
+Daytona allows you to delete Targets, helping you manage and remove those that are no longer needed. Once a Target is deleted, you will not be able to manage or deploy Workspaces without adding another Target with the same configuration.
```

Comment:
> Once a Target is deleted, you will not be able to manage or deploy Workspaces without adding another Target with the same configuration.

This is a bit harsh. You can manage workspaces on other targets but this sentence makes it seem like your completely blocked if you remove any target.

I suggest rephrasing so its clear that you can not manage workspace on that target only.